src/test/java/com/dao/OrderDAOTest.java

```diff
@@ -0,0 +1,105 @@
+package com.dao;
+
+import com.model.Order;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabase;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseBuilder;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+
+import static org.junit.Assert.*;
+import static org.hamcrest.CoreMatchers.*;
+
+import javax.sql.DataSource;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+
+
+public class OrderDAOTest {
+
+    private OrderDAO orderDAO = OrderDAO.getInstance();
+
+    @Before
+    public  void init() {
+        orderDAO.setDataSource(getDataSource());
+    }
+
+
+    @Test
+    public void getAllOrders() throws Exception {
+        List<Order> actual = orderDAO.getAll();
+
+        List<Order> expected = Arrays.asList(
+                new Order(1,1,LocalDateTime.of(2017,10,25,13,54,19),475,Order.status.CREATED),
+                new Order(2,2,LocalDateTime.of(2017,10,25,13,54,19),290,Order.status.CONFIRMED),
```

Comment:
too many arguments in one line
try to make it more readable by splitting in a few lines